concepticondata/concepticon.tsv

```diff
@@ -3142,3 +3142,11 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3146	CHEETAH	Animals	Wild Animal of the cat family with characteristic dark spots.	Person/Thing	
 3147	HAMSTER	Animals	Mouse-sized animal with large cheeks for storing food, often kept as a pet.	Person/Thing	
 3148	MEERKAT	Animals	Southern African animal known for its long tail and often seen standing up on its back legs.	Person/Thing	
+3149	ARMLET	Clothing and grooming	A band or bracelet worn round the upper part of a person's arm.	Person/Thing	
+3150	BOX OR CHEST	Modern world	Either a generic manmade object with space in it that is used to keep and store many various different objects, or a strong box with a lid on top.	Property	
+3151	CUT (WITH AXE)	Basic actions and technology	To slice or to divide into pieces with help of an axe.	Person/Thing	
+3152	MUNTJACS	Animals	Also known as barking deer, a kind of small deer of the genus Muntiacus.	Person/Thing	
+3153	LORD	Social and political relations	A man of noble rank or high office; a nobleman.	Person/Thing	
+3154	WILD BOAR	Animals	Also known as wild pig, four-legged animal (Sus scrofa) living in the wild. 	Person/Thing	
```

Comment:
we have the BOAR, and BOAR often refers to wild boar, so ideally check that the definition is exactly that of a wild boar (in boar), to make sure we don't add another wild boar to our concepticon. If boar is "male pig", leave wild boar...

Reply:
boar in the concepticon list is  "An adult male pig."  I think it is a broad category. 
In Marrison's data, he used "pig (wild)", so I think it might be good to add "wild boar" to specify the pig is not in captivity, and not a domestic animal. Also, it is not equal to feral pig, which describes the descended from escaped domesticated pig. 
I will look into Marrison's thesis and see if he has further description about this lexical item.

Reply:
This means I might have mismapped concepts in Sun's ZYYMC concept list -- if I remember, I also mapped to BOAR assuming it was WILD BOAR. Need to check.